filecheck/checker.ts

```diff
@@ -33,14 +37,36 @@ interface CheckerOptions {
   saveCompression?: boolean;
 }
 
+class FixableError extends Error {
+  fixCommand: string;
+
+  constructor(message: string, fixCommand: string) {
+    super(message);
+    this.fixCommand = fixCommand;
+  }
+
+  toString() {
+    return `${this.constructor.name}: ${this.message}`;
+  }
+}
+
+function getRelativePath(filePath: string): string {
+  return path.relative(process.cwd(), filePath);
+}
+
 export async function checkFile(
   filePath: string,
   options: CheckerOptions = {}
 ) {
   // Check that the filename is always lowercase.
-  if (path.basename(filePath) !== path.basename(filePath).toLowerCase()) {
-    throw new Error(
-      `Base name must be lowercase (not ${path.basename(filePath)})`
+  const expectedPath = path.join(
+    path.dirname(filePath),
+    path.basename(filePath)
+  );
+  if (filePath !== expectedPath) {
+    throw new FixableError(
+      `Base name must be lowercase (not ${path.basename(filePath)})`,
+      `mv '${getRelativePath(filePath)} '${getRelativePath(expectedPath)}'`
```

Comment:
Will this actually call `mv`? That won't work on all Windows shells. PowerShell will alias that command, but not the regular `cmd` environment